### ISSUE TITLE
Support dmenu -mask option

### ISSUE THREAD

**Issue by s-ol** [firecat53/keepmenu#46]
`-mask` in dmenu is like `-password` for rofi.

**Comment by firecat53** [comment]
Same idea is already supported: see config.ini.example:

    # # override normal foreground and background colors (dmenu) or use the
    # # -password option (rofi) to obscure passphrase entry
    # [dmenu_passphrase]
    # nf = #222222
    # nb = #222222
    # rofi_obscure = True

**Comment by s-ol** [comment]
exactly, but none of the three parameters can be used to make it work with dmenu natively.

**Comment by firecat53** [comment]
Maybe I'm misunderstanding something...dmenu doesn't have a `-mask` option. What are you trying to accomplish?

**Comment by firecat53** [comment]
Did you try just adding -mask to the dmenu_command line in config.ini? It should support command line arguments.

    [dmenu]
    dmenu_command = /usr/bin/dmenu2 -mask

**Comment by s-ol** [comment]
that also obscures search input in normal use. If there was a `dmenu_passphrase.dmenu_command` that would work.

**Comment by firecat53** [comment]
Ah, yeah, makes sense. I'll try and figure out the best way to handle it. May take awhile....not much development time lately :(

**Comment by s-ol** [comment]
No worries, I am using nf and nb for now, just wanted to mention this here in case you didn't know of the option ;)
